audio.js

```diff
@@ -1,12 +1,225 @@
+                            keySounds(113, "Heater 1");
+                            keySounds(119, "Heater 2");
+                            keySounds(101, "Heater 3");
+                            keySounds(97, "Heater 4");
+                            keySounds(115, "Clap");
+                            keySounds(100, "Open HH");
+                            keySounds(122, "Kick Hat");
+                            keySounds(120, "Kick");
+                            keySounds(99, "Closed HH");
+                        });
+                    }
+                    break;
+                case '1':
+                    // console.log('bank.value == 1');
+                    if (toggle.value == '1') {
+                        $(document).keypress(function (event) {
+                            // console.log(event.keyCode);
+                            keySounds(113, "Chord 1");
+                            keySounds(119, "Chord 2");
+                            keySounds(101, "Chord 3");
+                            keySounds(97, "Shaker");
+                            keySounds(115, "Open H");
+                            keySounds(100, "Closed H");
+                            keySounds(122, "Punchy Kick");
+                            keySounds(120, "Side Stick");
+                            keySounds(99, "Snare");
+                        });
+                    }
+                    break;
+                default:
+                    console.log('switch');
+            }
+        });
+
+        $(".playback").removeAttr('disabled'); //audio on
+
+        $(".playback").click(function (e) {
+            e.preventDefault();
+
+            var clip = $(this).next('audio').get(0);
+
+            if (clip === undefined) {
+                display.innerHTML = '';
+            } else {
+                display.innerHTML = clip.id;
+                clip.currentTime = 0;
+
+                let playPromise = clip.play();
+
+                if (playPromise !== undefined) {
+                    playPromise.then(_ => {
+                        clip.play();
+                    })
+                        .catch(error => {
+                            console.log(error);
+                        })
+                }
+            }
+        });
+    } else {
+        toggle.value = '0';
+        display.innerHTML = 'Power Off';
+        $(document).unbind('keyup keydown keypress'); //keyboard off
+        $(".playback").attr('disabled', 'disabled'); // audio off
+    }
+});
+
+//Bank ON and OFF
+let bank = document.getElementById("bank-switch-input");
+
+// console.log(typeof bank.value);
+
+bank.addEventListener('input', (event) => {
+    event.preventDefault();
+
+    if (bank.value == '0') {
+        bank.value = '1';
+
+        display.innerHTML = 'Smooth Piano Kit';
+
+        let newQ = " <audio id='Chord 1' class='chord1' src='https://s3.amazonaws.com/freecodecamp/drums/Chord_1.mp3'></audio>";
```

Comment:
Personally, I'm not a huge fan of putting HTML code in JS files or vice versa. But sometimes it just happens. Frankly, there's a hundred different ways to handle this problem and I have no idea what's the "right" way to solve it. I'll make my suggestion, send it to you later, and we can talk about the pros and cons of each at that time.

Reply:
HTML hot swap on button press was the quickest solution I came up with to avoid dom errors and to avoid putting another set of audio elements in the index.html. Button clicks and key presses correctly associate to the correct audio element this way.